config/database.yml

```diff
@@ -16,8 +16,8 @@
 #
 default: &default
   adapter: sqlite3
+  encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
 
 development:

```

Comment:
- [ ] Kindly make sure you are using a PostgreSQL database as it's required for this project.